### PR TITLE
feat: multiselect 

### PR DESCRIPTION
**Pull request by fsansalvadore**
- [x] full a11y
- [x] composable combobox -> add MultiSelectorInput to show the search input
- [x] optional badgeLimit={number | "wrap"} prop -> shows "+1", "+2" if setting a number, or wraps the badges to new line. Default is they overflow-hidden and are scrollable horizontally.
- [x] optional showIcon to show/hide ChevronsUpDown icon
- [x] optional deletableBadge to enable/disable removing badges directly
- [x] optional persistLabel to keep showing the placeholder label 
- [x] add the "inline search input" as previous multi-select by adding mode="inline-combobox" to the trigger
- [x] reset input value when adding item via combobox

### CODE REVIEW

**Review by jordienr — APPROVED** [review]

File: apps/docs/features/docs/Troubleshooting.ui.client.tsx [hunk 1]
```diff
@@ -195,112 +200,61 @@ function TroubleshootingFilterInternal({
...
-    [_setTagsOpen]
-  )
-
   const convertToItem = useCallback(
     (arr: string[], { capitalize = false }: { capitalize?: boolean } = {}) =>
-      arr.map((item) => ({
-        value: item,
-        label: capitalize ? item[0].toUpperCase() + item.slice(1) : item,
-      })),
+      arr.map((item) => item),
```
[the remark this answers is not in the document]
Comment (fsansalvadore) [reply apps/docs/features/docs/Troubleshooting.ui.client.tsx:205]: checking

File: packages/ui/src/components/shadcn/ui/command.tsx [hunk 2]
```diff
@@ -41,20 +41,46 @@ const CommandInput = React.forwardRef<
...
+      {showSearchIcon && <Search className="h-4 w-4 shrink-0 opacity-50" />}
+      <CommandPrimitive.Input
+        ref={ref}
+        className={cn(
+          'flex h-9 w-full rounded-md bg-transparent py-3 text-xs text outline-none placeholder:text-muted disabled:cursor-not-allowed disabled:opacity-50 border-none focus:ring-0',
+          className
+        )}
+        {...props}
+      />
+      {showCloseIcon && (
```
Comment (MildTomato) [inline packages/ui/src/components/shadcn/ui/command.tsx:70]: What's this for ?

Comment (fsansalvadore) [reply packages/ui/src/components/shadcn/ui/command.tsx:70]: It's this x to reset the input. Wrong naming now that you mention it...

https://github.com/user-attachments/assets/9b8c748d-868a-4a7a-ac1b-089d01025117

Comment (fsansalvadore) [reply packages/ui/src/components/shadcn/ui/command.tsx]: `showResetIcon` now

File: apps/docs/features/docs/Troubleshooting.ui.client.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (fsansalvadore) [reply apps/docs/features/docs/Troubleshooting.ui.client.tsx]: removed 👍
it was leftover code, thanks

**Pull request merged**